assets/js/filter_players_list.js

```diff
@@ -0,0 +1,18 @@
+const handleSportChange = e => {
+  const allTables = document.querySelectorAll(".fantasy-player-collection");
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)